<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>marcas</class>
 <widget class="QDialog" name="marcas">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seleccion de marcas</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableView" name="tablaMarcas">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="btmAgregarMarca">
       <property name="toolTip">
        <string>Agregar Marca</string>
       </property>
       <property name="whatsThis">
        <string>Puede agregar nuevas marcas</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/mas.png</normaloff>:/iconos/mas.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btmEliminarMarca">
       <property name="toolTip">
        <string>Eliminar Marca</string>
       </property>
       <property name="whatsThis">
        <string>Elimine las marcas ya no utilizadas</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/menos.png</normaloff>:/iconos/menos.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btmModificarMarca">
       <property name="toolTip">
        <string>Modificar marca</string>
       </property>
       <property name="whatsThis">
        <string>Cambie el nombre de la marca</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/default-applications-capplet.png</normaloff>:/iconos/default-applications-capplet.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelMarcas">
     <property name="text">
      <string>Marcas:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTableView" name="tablaModelo">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelModelos">
     <property name="text">
      <string>Modelos:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QToolButton" name="btmAgregarModelo">
       <property name="toolTip">
        <string>Agregar modelos</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/mas.png</normaloff>:/iconos/mas.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btmEliminarModelo">
       <property name="toolTip">
        <string>Eliminar modelos</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/menos.png</normaloff>:/iconos/menos.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btmModificarModelo">
       <property name="toolTip">
        <string>Modificar modelos</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/default-applications-capplet.png</normaloff>:/iconos/default-applications-capplet.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btmSeleccionar">
       <property name="text">
        <string>Seleccionar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/button_ok.png</normaloff>:/iconos/button_ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btmCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/exit.png</normaloff>:/iconos/exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="insertRowAction">
   <property name="text">
    <string>Insertar Fila</string>
   </property>
   <property name="toolTip">
    <string>Insertar Fila</string>
   </property>
  </action>
  <action name="deleteRowAction">
   <property name="text">
    <string>Delete Row</string>
   </property>
  </action>
  <action name="modificarRowAction">
   <property name="text">
    <string>modficiar row</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
